Codeinwp/visualizer
# Load necessary google charts packages in Gutenberg

## Summary 

I added the necessary packages as parameters for the Charts component from React Google Chart lib.

## Notes

This took some time to debug. At first, it might be a problem with the given data to the component since it throws an undefined error. After some extensive testing, it was not the case.

After reviewing the documents, I found that packages might not be loaded (like `geochart`). After trying a bunch of things, I looked more deeply at the lib source code and found this default:

 https://github.com/rakannimer/react-google-charts/blob/418ebc924f36ba32eea48d79034a418042139c9e/src/hooks/useLoadGoogleCharts.ts#L23

By default, the lib does not load all necessary packages for us (like [here ](https://github.com/Codeinwp/visualizer/blob/6c2ad2d4ea458cdd85f60cd717d4d1cfd172742b/js/render-google.js#L546))

## Screenshots

![image](https://github.com/Codeinwp/visualizer/assets/17597852/bd00ebc0-a969-4fcd-bc79-8939b5033bd2)

## Testing

1. Make a default Geo Chart
2. Insert in the Gutenberg Editor via block
3. It should display
